### PR TITLE
Clean up the command documentation some more.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by peternewman** [comment]
Fix Travis.

File: firmware/src/constants.h [hunk 1]
```diff
@@ -88,25 +88,75 @@
  * @brief The Ja Rule message commands.
  */
 typedef enum {
-  ECHO = 0x80,  //!< Echo the data back
-  TX_DMX = 0x81,  //!< Transmit a DMX frame
+  ECHO = 0x80,  //!< Echo the data back. See @ref message_cmd_echo
+
```
Comment (peternewman) [inline firmware/src/constants.h]: Any reason for the space?

File: firmware/src/constants.h [hunk 2]
```diff
@@ -88,25 +88,75 @@
...
+  TX_DMX = 0x81,  //!< Transmit a DMX frame. See @ref message_cmd_txdmx.
   GET_LOG = 0x82,  //!< Fetch more log data
   GET_FLAGS = 0x83,  //!< Get the flags state
   WRITE_LOG = 0x84,
+
+  /**
+   * @brief Reset the device.
+   * See @ref message_cmd_reset.
```
Comment (peternewman) [inline firmware/src/constants.h]: I think I asked about @see before?

File: tests/doxygen/message-format.dox [hunk 3]
```diff
@@ -46,7 +46,7 @@
  * @param Token A token for the command. The same token will be returned in the
  * command response. Typically the host will increment the token with each
  * request.
- * @param Command The command ID.
+ * @param Command The \ref Command identifier.
```
Comment (peternewman) [inline tests/doxygen/message-format.dox]: Any reason for these not being @ref?

**Comment by peternewman** [comment]
LGTM

File: firmware/src/constants.h (hunk 2 above)
Comment (nomis52) [inline firmware/src/constants.h]: The enums are one per line. If I add @see it breaks the second sentence out into a new paragraph and the table gets much bigger.

File: tests/doxygen/message-format.dox (hunk 3 above)
Comment (nomis52) [inline tests/doxygen/message-format.dox]: Switched.

File: firmware/src/constants.h (hunk 1 above)
Comment (nomis52) [inline firmware/src/constants.h]: Sorry which space?

File: firmware/src/constants.h (hunk 2 above)
Comment (peternewman) [inline firmware/src/constants.h]: Okay, fair enough.

File: firmware/src/constants.h (hunk 1 above)
Comment (peternewman) [inline firmware/src/constants.h]: The random new space after echo. They don't seem to be grouped into logical groups, just randomly.

Comment (nomis52) [inline firmware/src/constants.h]: Oh, yeah, that's coming in a follow up . I'm going to group the enums and set the correct values.

**Pull request merged**